primitives/src/targeting/eval.rs

```diff
@@ -401,6 +519,191 @@ fn eval(input: &Input, output: &mut Output, rule: &Rule) -> Result<Option<Value>
 
             Some(Value::Bool(a && b))
         }
+        Function::Or(first_rule, second_rule) => {
+            let a = eval(input, output, first_rule)?
+                .ok_or(Error::TypeError)?
+                .try_bool()?;
+            let b = eval(input, output, second_rule)?
+                .ok_or(Error::TypeError)?
+                .try_bool()?;
+
+            Some(Value::Bool(a || b))
+        }
+        Function::Xor(first_rule, second_rule) => {
+            let a = eval(input, output, first_rule)?
+                .ok_or(Error::TypeError)?
+                .try_bool()?;
+            let b = eval(input, output, second_rule)?
+                .ok_or(Error::TypeError)?
+                .try_bool()?;
+
+            Some(Value::Bool((a && !b) || (!a && b)))
```

Comment:
Forgot about this last time, but you can use the XOR (`^`) operator instead, see:
https://doc.rust-lang.org/std/primitive.bool.html#impl-BitXor%3Cbool%3E